hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDecommissionWithStriped.java

```diff
@@ -462,6 +462,59 @@ public void testFileChecksumAfterDecommission() throws Exception {
         fileChecksum1.equals(fileChecksum2));
   }
 
+  /**
+   * Test decommission when DN marked as busy.
+   * @throwsException
+   */
+  @Test(timeout = 120000)
+  public void testBusyAfterDecommissionNode() throws Exception {
+    byte busyDNIndex = 0;
```

Comment:
Any consideration when define byte type for index here? Not blocker just out of interest.

Reply:
Thanks @Hexiaoqiao for you comment.
there is no special meaning to define byte type, maybe we can change it to int type.